src/scss/components/_sidenav.scss

```
@@ -34,3 +30,17 @@
   padding-top: units(1.5);
   padding-bottom: units(1.5);
 }
+
+.usa-sidenav__sublist .usa-current {
+  @include remove-bar;
+}
+
+.usa-sidenav__sublist .usa-sidenav__item {
+  border-top: 0;
+}
+
+.usa-sidenav .usa-current {
+  &:hover {
+      color: #205493;
```

Can we use the [corresponding color token](https://design.login.gov/color/) instead of hardcoding the hex?

```suggestion
      color: color('primary-dark');
```